website/docs/migration_guide.md

```diff
@@ -56,11 +64,13 @@ spark-submit --master local \
 --hoodie-conf hoodie.bootstrap.keygen.class=org.apache.hudi.keygen.SimpleKeyGenerator \
 --hoodie-conf hoodie.bootstrap.full.input.provider=org.apache.hudi.bootstrap.SparkParquetBootstrapDataProvider \
```

Comment:
I don't think we need `hoodie-conf hoodie.bootstrap.full.input.provider` in the example

Reply:
Fixed!